bootstrap.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/sh
 
+set -e
+
 echo Building the build tool...
 
 GOBIN="$(pwd)/util" go get -u harvey-os.org/cmd/ufs
 GO111MODULE=on GOBIN="$(pwd)/util" go get ./util/src/harvey/cmd/...
 
+echo Fetching u-root and building it...
+GO111MODULE=off go get github.com/u-root/u-root
+GO111MODULE=off go install github.com/u-root/u-root
```

Comment:
without the -d flag go get installs already. So we can collapse the two lines in to one
```suggestion
GOBIN="$(pwd)/util"  GO111MODULE=off go get github.com/u-root/u-root
```
I agree with @orangecms we should stick to keeping these local.